### PR TITLE
ANN Classification #4

### PR DESCRIPTION
**Pull request by NeonKazuha**
Used Artificial Neural Networks using tensorflow and keras to predict the values of 'Type' feature in the dataframe.
Uses several layers such as Dense, Dropout, BatchNormalization.
Sequential used to build the model structure.
Tested with validation Data as well.

### CODE REVIEW

**Comment by NeonKazuha** [comment]
@darshbaxi Please review.

**Comment by Samcoding5854** [comment]
Excellent work in the notebook. Really appreciate the markdown cells and each graphs and visualisations made. 
Could have tried to increase the accuracy by experimenting with neurons. If did then great work! 
Keep it up
